firewood/src/merkle.rs

```diff
@@ -1715,18 +1717,32 @@ impl<S: ShaleStore<Node> + Send + Sync, T> Merkle<S, T> {
         self.store.flush_dirty()
     }
 
-    pub(crate) fn iter(&self, root: DiskAddress) -> MerkleKeyValueStream<'_, S, T> {
+    pub(crate) fn key_value_iter(&self, root: DiskAddress) -> MerkleKeyValueStream<'_, S, T> {
         MerkleKeyValueStream::new(self, root)
     }
 
-    pub(crate) fn iter_from(
+    pub(crate) fn key_value_iter_from(
         &self,
         root: DiskAddress,
         key: Box<[u8]>,
     ) -> MerkleKeyValueStream<'_, S, T> {
         MerkleKeyValueStream::from_key(self, root, key)
     }
 
+    #[allow(dead_code)]
```

Comment:
I thought it made sense to expose this new functionality even though it's not used right now, but I can remove it since it's not used right now

Reply:
Hmm if I remove `#[allow(dead_code)]` then it complains that the code is unused, but if I remove the function, I can't test the node iterator, which I should do because it's used by the key-value iterator

Reply:
Mark them #[cfg(test)] if they are only used in testing, or perhaps only have them defined in the test module.

Reply:
moved to test module and did a bit of renaming